pages/addressBookPage.js

```diff
@@ -0,0 +1,353 @@
+ * @returns {Promise<void>} Clicks a button that adds new address.
+ */
+const clickAddNewAddress = async (page) => {
+    const optionElement = await page.waitForXPath(SELECTORS.ADD_NEW_ADRESS, {
+        visible: true,
+        timeout: COMMON_TIMEOUT
+    });
+    await optionElement.click();
+};
+
+/**
+ * Adds first name to first name bracket.
+ * @param page Represents page object of currently handled tab.
+ * @param firstName Chosen First Name.
+ * @returns {Promise<string>} Fills up First Name bracket with chosen First Name.
+ */
+const setFirstNameDetail = async (page, firstName = faker.name.firstName('male')) => {
+    const firstNameInput = await page.waitForXPath(SELECTORS.ADDRESS_DETAILS.FIRSTNAME_INPUT, {
+        visible: true,
+        timeout: COMMON_TIMEOUT
+    });
+    await firstNameInput.click({clickCount: 3})
+    await firstNameInput.type(firstName);
+
+    return firstName;
+};
+
+/**
+ * Set Last Name input field.
+ * @param page Represents page object of currently handled tab.
+ * @param lastName Chosen Last Name.
+ * @returns {Promise<string>} Fills up Last Name bracket with chosen Last Name.
+ */
+const setLastNameDetail = async (page, lastName = faker.name.lastName('male')) => {
+    const lastNameInput = await page.waitForXPath(SELECTORS.ADDRESS_DETAILS.LASTNAME_INPUT, {
+        visible: true,
+        timeout: COMMON_TIMEOUT
+    });
+    await lastNameInput.click({clickCount: 3})
+    await lastNameInput.type(lastName);
+
+    return lastName;
+};
+
+/**
+ * Adds Phone Number to Phone Number bracket.
+ * @param page Represents page object of currently handled tab.
+ * @param phoneNumber Chosen Phone Number.
+ * @returns {Promise<string>} Fills up Phone Number bracket with chosen Phone Number.
+ */
+const setPhoneNumberDetail = async (page, phoneNumber = faker.phone.number()) => {
+    const phoneNumberInput = await page.waitForXPath(SELECTORS.ADDRESS_DETAILS.PHONE_NUMBER_INPUT, {
+        visible: true,
+        timeout: COMMON_TIMEOUT
+    });
+    await phoneNumberInput.click();
+    await phoneNumberInput.type(phoneNumber);
+
+    return phoneNumber;
+};
+
+/**
+ * Adds Street Address to Street Address bracket.
+ * @param page Represents page object of currently handled tab.
+ * @param addressDetail Chosen Street Address.
+ * @returns {Promise<string>} Fills up Street Address bracket with chosen Street Address.
+ */
+const setStreetAddressDetail = async (page, addressDetail = faker.address.streetAddress(true)) => {
+    const streetAdressInput = await page.waitForXPath(SELECTORS.ADDRESS_DETAILS.STREET_ADDRESS_INPUT, {
+        visible: true,
+        timeout: COMMON_TIMEOUT
+    });
+    await streetAdressInput.click();
+    await streetAdressInput.type(addressDetail);
+
+    return addressDetail;
+};
+
+/**
+ * Adds City to City bracket.
+ * @param page Represents page object of currently handled tab.
+ * @param cityDetail Chosen City.
+ * @returns {Promise<string>} Fills up City bracket with chosen City.
```

Comment:
Remove types of returned value and change returns description. It must be clear what value you can expect from return statement of the function:
`* @returns cityDetail text.`